doc/reduce-traffic.md

```diff
@@ -3,8 +3,9 @@ Reduce Traffic
 
 Some node operators need to deal with bandwidth caps imposed by their ISPs.
 
-By default, Bitcoin Core allows up to 125 connections to different peers, 11 of
-which are outbound. You can therefore, have at most 114 inbound connections.
+By default, Bitcoin Core allows up to 200 connections to different peers, 11 of
+which are outbound. You can therefore, have at most 189 inbound connections, half of
+which can only be taken up by low-traffic block-relay-only peers.
```

Comment:
nit: I’m trying to understand something. According to the PR description, 50% of inbound slots are allocated for tx-relaying peers, but the commit message seems to imply that 50% are for block-relay-only peers. Does this mean that the inbound slots are equally split between tx-relaying and block-relay-only peers? If so, could the description be expanded for clarity?

Reply:
What is meant with is that half of the slots can only be accessed by block-relay-only peers, and  the other half is accessible to both:
"with 50% of inbound slots accessible for tx-relaying peers" and "half of
which can **only** be taken up by low-traffic block-relay-only peers" are therefore both saying the same thing:

If we'd have 0 tx-relay inobund peers, in theory all inbound slots could be filled up with block-relay-only peers, whereas if we had 0 block-relay-only inbound peers, still only half of the slots could be filled with tx-relay peers.